examples/network_peer/README.md

```diff
@@ -1361,4 +1361,829 @@ couchbase-capella_network_peer.new_network_peer: Destruction complete after 14s
+              + name            = "VNETPeerTFTestAZURE"
+              + provider_config = {
+                  + aws_config   = null
+                  + azure_config = {
+                      + cidr            = "10.6.0.0/16"
+                      + provider_id     = ""
+                      + resource_group  = "\"test-rg\""
+                      + subscription_id = "\"ffffffff-aaaa-1414-eeee-00000000000\""
+                      + tenant_id       = "\"ffffffff-aaaa-1414-eeee-00000000000\""
+                      + vnet_id         = "\"test-vnet\""
+                    }
+                  + gcp_config   = null
+                }
+              + status          = {
+                  + reasoning = ""
+                  + state     = "failed"
+                }
+            },
+        ]
+      + organization_id = "ffffffff-aaaa-1414-eeee-00000000000"
+      + project_id      = "ffffffff-aaaa-1414-eeee-00000000000"
+    }
+  + new_network_peer   = {
+      + audit           = (known after apply)
+      + cluster_id      = "ffffffff-aaaa-1414-eeee-00000000000"
+      + commands        = (known after apply)
+      + id              = (known after apply)
+      + name            = "VNETPeerTFTestAZURE"
+      + organization_id = "ffffffff-aaaa-1414-eeee-00000000000"
+      + project_id      = "ffffffff-aaaa-1414-eeee-00000000000"
+      + provider_config = {
+          + aws_config   = null
+          + azure_config = {
+              + cidr            = "10.6.0.0/16"
+              + provider_id     = (known after apply)
+              + resource_group  = "test-rg"
+              + subscription_id = "ffffffff-aaaa-1414-eeee-00000000000"
+              + tenant_id       = "ffffffff-aaaa-1414-eeee-00000000000"
+              + vnet_id         = "test-vnet"
+            }
+          + gcp_config   = null
+        }
+      + provider_type   = "azure"
+      + status          = (known after apply)
+    }
+  + peer_id            = (known after apply)
+
+Do you want to perform these actions?
+  Terraform will perform the actions described above.
+  Only 'yes' will be accepted to approve.
+
+  Enter a value: yes
+
+couchbase-capella_network_peer.new_network_peer: Creating...
+couchbase-capella_network_peer.new_network_peer: Still creating... [10s elapsed]
+couchbase-capella_network_peer.new_network_peer: Still creating... [20s elapsed]
+couchbase-capella_network_peer.new_network_peer: Still creating... [30s elapsed]
+couchbase-capella_network_peer.new_network_peer: Still creating... [40s elapsed]
+couchbase-capella_network_peer.new_network_peer: Creation complete after 43s [id=ffffffff-aaaa-1414-eeee-00000000000]
+
+Apply complete! Resources: 1 added, 0 changed, 0 destroyed.
+
+Outputs:
+
+network_peers_list = {
+  "cluster_id" = "ffffffff-aaaa-1414-eeee-00000000000"
+  "data" = tolist([
+    {
+      "audit" = {
+        "created_at" = "2024-08-23 23:30:39.790107885 +0000 UTC"
+        "created_by" = "s10qrydhDaic9GsWZ6BYpJfSK7wARDCv"
```

Comment:
should we also change this to some generic apikey?

Reply:
For this network peers I just substituted it with `sample_apikey` but we need to update this in all readmes